src/main/java/uk/gov/hmcts/darts/ws/DartsEndpoint.java

```diff
@@ -46,79 +51,109 @@ public class DartsEndpoint {
 
     @PayloadRoot(namespace = "http://com.synapps.mojdarts.service.com", localPart = "addDocument")
     @ResponsePayload
-    public JAXBElement<AddDocumentResponse> addDocument(@RequestPayload JAXBElement<AddDocument> addDocument) {
+    public JAXBElement<AddDocumentResponse> addDocument(
```

Comment:
Do we need to modify the endpoints at all. We can now handle this in isolation through a servlet filter?

Reply:
Yes, I struggled with the current DartsEndpoint(s) in general - the only success I had was to make use of what we see here, using the @SoapHeader annotation alongside the @RequestPayload (spring-ws-core).

Reply:
Done in commit 394fb93 - Replace @SoapHeader in DartsEndpoint and AuthenticationAspect with SoapRequestInterceptor